chromiumcontent/BUILD.gn

```diff
@@ -110,41 +108,11 @@ if (is_electron_build && is_component_build) {
     public_deps = [ "//services/device:lib" ]
   }
 
-  static_library("fx_agg") {
```

Comment:
how come we no longer need this?

Reply:
We do need it, but its already built by pdfium as a dependency https://github.com/electron/libchromiumcontent/pull/575/files#diff-6bcdc4060f1a8322ee9b67ee3fcc7078R105, this step is not needed.

Reply:
👍